endpoints/cookie_sync.go

```diff
@@ -132,10 +138,8 @@ func (c *cookieSyncEndpoint) parseRequest(r *http.Request) (usersync.Request, pr
 		}
 	}
 
-	request, err = c.setLimit(request)
-	if err != nil {
-		return usersync.Request{}, privacy.Policies{}, err
-	}
+	request = c.setLimit(request, account.CookieSync)
+	request = c.setCooperativeSync(request, account.CookieSync)
```

Comment:
Yes it will never be nil. The `account.account.go#GetAccount` function called above this line always returns either an account object or an error. We rely on this behavior in our auction flows as well. I think the `GetAccount` unit test does a good job of ensuring this is the case. I suppose we could add an `if account != nil...return` check just after calling `GetAccount` here and in each of the auction flows to be safe. Alternatively we could make `GetAccount` always return an account object, in which case the function would need to be adjusted to still return an account object even when there is an error...or we could just leave everything as is. Thoughts?